h2o-r/demos/rdemo.airlines.target_encode.R

```diff
@@ -0,0 +1,130 @@
+library(h2o)
```

Comment:
please check if we need it

Reply:
I've cleaned it up a little bit so I think we can keep it.